test cases/common/234 very long commmand line/seq.py

```diff
@@ -1,6 +1,19 @@
 #!/usr/bin/env python3
+"""
+generate sequence of filename that does not exceed MAX_LEN=260
+for Python < 3.6 and Windows without modified registry
+"""
 
 import sys
+import string
 
-for i in range(int(sys.argv[1]), int(sys.argv[2])):
-    print(i)
+name_len = 260 - len(sys.argv[2]) - 4 - 39 - 4 - 2
```

Comment:
It seems like this could be zero or less, if `meson.build_root()` is long enough.  This test can't work in that case? (or if it's small enough that we can't generate enough unique filenames?)